pages/windows/assoc.md

```diff
@@ -14,3 +14,7 @@
 - Set the associated file type for a specific extension:
 
 `assoc .{{txt}}={{txtfile}}`
+
+- View the output of assoc one screen at a time:
+
+`assoc | more`
```

Comment:
Not sure about this example, pipe with `more` could be added to almost all the pages.

Reply:
I think this example is unnecessary as well.

Reply:
> I think this example is unnecessary. Piping to `more` is a well-known technique.

Yeah, But this page has 3 commands only so adding this won't be an issue I guess.